test/nn/test_model_summary.py

```diff
@@ -195,8 +195,9 @@ def test_summary_with_to_hetero_model():
     }
     metadata = list(x_dict.keys()), list(edge_index_dict.keys())
     model = to_hetero(GraphSAGE(), metadata)
-
-    expected = """
+    # flake8: noqa
+    if not WITH_TO_HETERO_HETEROLIN:
```

Comment:
I think we should run both test cases by mocking `WITH_TO_HETERO_HETEROLIN` ideally

Reply:
i am not sure i understand your comment. This flag exists so that when we are in an environment where we don't use ToHeteroLinear for to_hetero, we should expect model summary to match the original ModuleDict based implementation

Reply:
Yes, and I was just suggesting we should be able to run both of the two test cases regardless of whether the package `pyg-lib` is actually installed by mocking some components including the flag, but feel free to ignore my comment as we can maybe do this in a future PR.

Reply:
yes i think lets focus on this in a future PR